### ISSUE TITLE
FR: Create skeleton directory

### ISSUE THREAD

**Issue by enbrnz** [owncloud-ansible/owncloud#25]
You can define a custom skeleton directory, there should be an extra variable like `owncloud_create_empty_skeleton`.

Additionally it would be cool if you could upload a folder from a local source or download a `tar.gz` from a webserver with another additional variable.

**Comment by xoxys** [comment]
There is already a variable 'owncloud_skeleton_path' why not use this one to create the dir?

**Comment by enbrnz** [comment]
Yes I know this one, that's what I was referring to with that you can define one. 

You think it's better to just give that variable some more functions? Could be an idea as well, I don't mind how to implement it.

**Comment by enbrnz** [comment]
Option to create the folder through [`Create ownCloud directories` in prepare.yml](https://github.com/owncloud-ansible/owncloud/blob/5e702d0c36d2fde95caa6fc3dbc26d4408530ac3/tasks/prepare.yml#L68).
Still open question: how to add specific files to it?
